governance/SIG Security Charter.md

```diff
@@ -1,97 +1,73 @@
 # SIG Security Charter
 
-This charter adheres to the Roles and Organization Management specified in <sig-governance>.
- Team information may be found in the <readme.md>
-
 ## Overview of SIG
-A central knowledge advisory group that provides resources and information security compliant, resolution and XYZ.
-
-All about security of systems and infrastructure resources. Security patching, resources and infra up to date.
-Provide the basic services of APPSEC related to this project.
-
-Two concise lines explaining what this SIG does with bullet points of the major responsibilities
-
-- Responsibility
+SIG Security is a special interest group for O3DE code security that works as a central knowledge advisory group and  provides resources, consultation, reporting mechanisms  and information for security compliance, and security related issue resolution. This SIG also helps in keeping security patching up to date, and provide the basic services related to  APPSEC (Application Security) of O3DE.
```

Comment:
nit: "is a special interest group for O3DE security" - remove code?

nit: "reporting mechanisms and" - looks like two spaces between "mechanisms" and "and"

Reply:
I feel like this paragraph would be more readable if we simplified the sentence structure. What do you think about:

"SIG Security is a special interest group for O3DE security. It serves as an advisory group for security related issues like compliance, security issue resolution, and security patching. It's also the group responsible for maintaining vulnerability reporting mechanisms and assisting with other tasks that fall under the [Application Security (AppSec)](https://en.wikipedia.org/wiki/Application_security) umbrella of concern."